src/formatting_stack/formatters/cljfmt.clj

```diff
@@ -14,6 +18,29 @@
                                  (cljfmt.main/fix [filename] {:indents indents})))))
   nil)
 
-(speced/defn new [{:keys [third-party-indent-specs] :as options}]
-  (implement (assoc options :id ::id)
-    formatter/--format! format!))
+(defn lint! [{:keys [third-party-indent-specs]} files]
+  (->> files
+       (process-in-parallel! (fn [filename]
+                               (let [indents (impl/cljfmt-indents-for filename third-party-indent-specs)
+                                     {{:keys [okay]} :counts
+                                      :keys [diff]} (#'cljfmt.main/check-one {:indents indents} filename)]
```

Comment:
Please don't invoke private memebers - f-s users can always bump `cljfmt` independently so `cljfmt` is not a dep which we control